### ISSUE TITLE
serialStrategy vendor property not working

### ISSUE THREAD

**Issue by smugleafdev** [MarathonLabs/marathon#348]
I've tried every single enum available for serialStrategy and nothing changes. It continues to only run on one of my three emulators.

Related to [this](https://github.com/Malinskiy/marathon/issues/223) issue and [this](https://github.com/Malinskiy/marathon/pull/231) enhancement.

```
$ adb devices
List of devices attached
emulator-5554	device
emulator-5556	device
emulator-5558	device
```

```
serialStrategy: "ddms"
```

```
D 10:32:23.270 [main @coroutine#8] <Scheduler> device EMULATOR30X0X0X0 associated with poolId omni
D 10:32:23.271 [main @coroutine#8] <Scheduler> pool actor omni is being created
D 10:32:23.368 [main @coroutine#8] <Scheduler> device EMULATOR30X0X0X0 associated with poolId omni
D 10:32:23.371 [main @coroutine#8] <Scheduler> device EMULATOR30X0X0X0 associated with poolId omni
D 10:32:23.373 [main @coroutine#9] <DevicePoolActor[omni]> add device EMULATOR30X0X0X0
W 10:32:23.383 [main @coroutine#9] <DevicePoolActor[omni]> device EMULATOR30X0X0X0 already present in pool omni
W 10:32:23.383 [main @coroutine#9] <DevicePoolActor[omni]> device EMULATOR30X0X0X0 already present in pool omni
```

EDIT: "marathon_property" seems to work. "ddms" does not.

**Comment by tagantroy** [comment]
Hello @smugleafdev, please try with latest version 0.5.2

**Comment by smugleafdev** [comment]
Fixed with 0.5.2!
